src/modules/manage-leaderboard.js

```diff
@@ -0,0 +1,15 @@
+export const onAddscore = (data) => data;
+
+export const render = (data, domContainer) => {
+  let domUI = '';
+  if (data.length > 0) {
+    data.forEach((item) => {
+      domUI += `<li class='score-item' id='${item.id}'>
+      ${item.name}: <span class="badge-score">${item.score}</span>
+      </li>`;
+    });
+  } else {
+    domUI = '<p class=\'no-item\'>There is not score to display</p>';
+  }
+  domContainer.innerHTML = domUI;
+};
```

Comment:
- [ ] While styling is not required, you are still expected to maintain the layout from the wireframe. Kindly fix your app layout.

  Your layout | Wireframe layout
  :---:|:---:
  ![dhhd](https://user-images.githubusercontent.com/54848363/208673064-f435b7b7-fba7-49b6-8029-ce5fea04abb6.jpg) | ![yydd](https://user-images.githubusercontent.com/54848363/208673053-1c0f7c75-0742-43a2-a95c-e3a98a4a9737.jpg)